corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -62,43 +62,73 @@
     </div>
     <% } %>
 
-    <% if (pageCount) { %>
+    <% if (endPage) { %>
     <div>
       <nav class="module-caselist-pagination-container">
         <ul class="pagination">
           <% if (currentPage > 0) { %>
+          <li class="page-item">
+            <a class="page-link" data-id="<%= 0 %>">
+              <span aria-hidden="true"><i class='fa fa-fast-backward'></i></span>
+              <span class="sr-only">{% trans "First page" %}</span>
+            </a>
+          </li>
+          <% } else { %>
+            <li class="paginate-disabled">
+              <a class="paginate-disabled">
+                <span aria-hidden="true"><i class='fa fa-fast-backward'></i></span>
+                <span class="sr-only">{% trans "First page (none)" %}</span>
+              </a>
+            </li>
+            <% } %>
+          <% if (currentPage > 0) { %>
           <li class="page-item" id="prev-page">
             <a class="page-link" aria-label="Previous" data-id="<%= currentPage - 1 %>">
```

Comment:
Could you translate the aria labels, too? You can use `trans_html_attr` instead of `trans` to make sure the value is escaped properly to be used as an attribute.